bin/arnold

```diff
@@ -487,6 +495,18 @@ function _docker_run() {
       -v ${COMMON_DIR}:/app/${RELATIVE_COMMON_DIR} \
       -v ${CUSTOMER_DIR}:/app/${RELATIVE_CUSTOMER_DIR}"
 
+    # Override default env_types if the local env_type dir exists
+    if [[ -e ${ENV_TYPE_DIR} ]]; then
+      extras_volumes="${extras_volumes} \
```

Comment:
The "s" in "extra" is confusing and I wondered why there is another ${ARNOLD_EXTRA_VOLUMES} variable with a similar name but holding something different.

From what I understand, this variable would be better named: "volumes" or "mounted_volumes". They are not the extras, they are all the volumes to mount.

Reply:
In fact the only required volume is the `${HOME}/.kube` I decided to consider all other volumes as "extra volumes". You are right about the typo, should be `extra_volumes`. The `ARNOLD_EXTRA_VOLUMES` var stores volumes passed to the CLI option `--extra-volume`.

Reply:
makes sense thanks for the clarification.